src/FFMpeg/Media/Video.php

```diff
@@ -53,72 +53,47 @@ public function addFilter(FilterInterface $filter)
      *
      * @param FormatInterface $format
      * @param string          $outputPathfile
+     * @param array           $additionalOptions
      *
      * @return Video
      *
      * @throws RuntimeException
      */
-    public function save(FormatInterface $format, $outputPathfile)
+    public function save(FormatInterface $format, $outputPathfile, array $additionalOptions = array())
```

Comment:
I don't like the idea to pass unfiltered stuff to the save method. We have an filter API..